### PR TITLE
chore: add sample function parameters, without type annotations

### PR DESCRIPTION
**Pull request by dizcology**
Part of https://github.com/googleapis/google-cloud-python/issues/16419, this PR adds the sample function's parameters as defined in the snippet config, but without type annotations.

### CODE REVIEW

File: gapic/configurable_snippetgen/configured_snippet.py [hunk 1]
```diff
@@ -79,12 +79,21 @@ def filename(self) -> str:
         sync_or_async = "sync" if self.is_sync else "async"
         return f"{module_name}_{self.api_version}_generated_{service_name}_{snake_case_rpc_name}_{config_id}_{sync_or_async}.py"
 
+    def _add_sample_funciton_parameters(self) -> None:
```
Comment (quartzmo) [inline gapic/configurable_snippetgen/configured_snippet.py:82]: typo: `funciton`

File: gapic/configurable_snippetgen/libcst_utils.py [hunk 2]
```diff
@@ -34,3 +34,26 @@ def base_function_def(function_name: str, is_sync: bool) -> libcst.FunctionDef:
...
+    value_name = config_expression.WhichOneof("value")
+    if value_name == "string_value":
+        string_value = config_expression.string_value
+        return libcst.SimpleString(value=f'"{string_value}"')
+    else:
+        raise ValueError(
+            f"Conversion from Expression value {value_name} unsupported.")
+
+
+def convert_parameter(
```
Comment (quartzmo) [inline gapic/configurable_snippetgen/libcst_utils.py:51]: It's interesting to see now how you are using LibCST!

File: gapic/configurable_snippetgen/configured_snippet.py (hunk 1 above)
Comment (dizcology) [reply gapic/configurable_snippetgen/configured_snippet.py]: fixed - thanks!

**Review by amanda-tarafa — APPROVED** [review]
This is great.
LGTM

**Review by atulep — APPROVED** [review]

**Pull request merged**